js/plugins/checks/src/evaluation.ts

```diff
@@ -77,11 +77,7 @@ export function checksEvaluators(
     }
   );
 
-  const evaluators = policy_configs.map((policy_config) => {
-    return createPolicyEvaluator(projectId, auth, ai, policy_config);
-  });
-
-  return evaluators;
+  return [createPolicyEvaluator(projectId, auth, ai, policy_configs)];
```

Comment:
Why not just make this return an `EvaluatorAction` instead of an array?

Reply:
done!